modules/oadp-mcg-performance-tuning.adoc

```diff
@@ -0,0 +1,139 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-mcg.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-mcg-performance-tuning_{context}"]
+= Performance tuning guide for Multicloud Object Gateway
+
+The default configuration of {rh-storage} for Multicloud Object Gateway is optimized for low resource consumption and not performance.
+
+
+* Large files: In the case of large files, the metadata-to-data ratio is low. Increasing the resources for the endpoints (memory and CPU) and the number of endpoints would be the first thing to do. In the case of namespace buckets, only memory would be sufficient. 
++
+The CPU is important mainly for data buckets, where the endpoints use the CPU for encryption and deduplication.
+
+* Small objects: In the case of small objects, the metadata-to-data ratio is high. For data buckets, this means a high level of resource involvement from the core and database. Increasing those pods resources would be the first step.
++
+Endpoints memory will probably not be under pressure if the core and database respond quickly. If they do not respond quickly enough, then pressure will build, and endpoints will eventually be under pressure. In this case, you should increase both core and database, with more emphasis on the database itself.
+
+* A high number of configuration entities, such as a large number of buckets or accounts: This would also point to the database and core, with more emphasis on the core than the database.
+
+When using namespace buckets, increasing the endpoint's memory and the database's memory and CPU would be the first step.
+
+*  MCG database resources: You need to increase CPU and memory based on the workload characteristics.
+
+* MCG auto scale min/max size - That would improve the response to peaks, but it also has a delay till it kicks in, hence it's important to set both minimal and maximum size.
```

Comment:
```suggestion
* MCG auto-scales the minimum and maximum size: This would improve the response to peaks, but it also has a delay until it reacts. Therefore, it is important to set both the minimum and maximum size.
```